package.json

```diff
@@ -8,19 +8,26 @@
     ]
   },
   "scripts": {
+    "cy:open": "npx cypress open",
+    "cy:run": "npx cypress run",
     "eslint": "eslint . --fix --ignore-path .gitignore --ext .ts,.tsx",
     "eslint:check": "eslint . --ignore-path .gitignore --ext .ts,.tsx",
     "prettier": "prettier --ignore-path .gitignore --write \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
-    "prettier:check": "prettier --ignore-path .gitignore --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\""
+    "prettier:check": "prettier --ignore-path .gitignore --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
+    "test-start-server": "jupyter lab --LabApp.token='test' --no-browser --port-retries=0",
```

Comment:
Where the lab server logs are being outputted in this case? If it's easily visible/accessible we could enable `--debug` to help diagnose errors